ui_framework/src/components/context_menu/context_menu.test.js

```diff
@@ -67,20 +72,24 @@ describe('KuiContextMenu', () => {
           .toMatchSnapshot();
       });
 
-      it('allows you to click the title button to go back to the previous panel', () => {
+      it('allows you to click the title button to go back to the previous panel', async () => {
         const component = mount(
           <KuiContextMenu
             panels={panels}
             initialPanelId={2}
           />
         );
 
+        await tick(20);
+
         expect(takeMountedSnapshot(component))
           .toMatchSnapshot();
 
         // Navigate to a different panel.
         component.find('[data-test-subj="contextMenuPanelTitleButton"]').simulate('click');
 
+        await tick(20);
```

Comment:
I think the `requestAnimationFrame` code in that component is the reason I had to do this.